opslevel/resource_opslevel_integration_google_cloud.go

```diff
@@ -0,0 +1,284 @@
+	TagsOverrideOwnership types.Bool                        `tfsdk:"ownership_tag_overrides"`
+}
+
+func newIntegrationGoogleCloudResourceModel(googleCloudIntegration opslevel.Integration, givenModel integrationGoogleCloudResourceModel) integrationGoogleCloudResourceModel {
+	resourceModel := integrationGoogleCloudResourceModel{
+		Aliases:     OptionalStringListValue(googleCloudIntegration.GoogleCloudIntegrationFragment.Aliases),
+		ClientEmail: givenModel.ClientEmail,
+		CreatedAt:   ComputedStringValue(googleCloudIntegration.CreatedAt.Local().Format(time.RFC850)),
+		Id:          ComputedStringValue(string(googleCloudIntegration.Id)),
+		InstalledAt: ComputedStringValue(googleCloudIntegration.InstalledAt.Local().Format(time.RFC850)),
+		Name:        RequiredStringValue(googleCloudIntegration.Name),
+		PrivateKey:  givenModel.PrivateKey,
+	}
+	if givenModel.OwnershipTagKeys.IsNull() {
+		resourceModel.OwnershipTagKeys = types.SetNull(types.StringType)
+	} else {
+		resourceModel.OwnershipTagKeys = StringSliceToSetValue(googleCloudIntegration.GoogleCloudIntegrationFragment.OwnershipTagKeys)
+	}
+	if givenModel.TagsOverrideOwnership.IsNull() {
+		resourceModel.TagsOverrideOwnership = types.BoolNull()
+	} else {
+		resourceModel.TagsOverrideOwnership = types.BoolValue(googleCloudIntegration.GoogleCloudIntegrationFragment.TagsOverrideOwnership)
+	}
+
+	projects := make([]googleCloudProjectResourceModel, len(googleCloudIntegration.Projects))
+	for i, project := range googleCloudIntegration.Projects {
+		projects[i] = googleCloudProjectResourceModel{
+			ID:   RequiredStringValue(project.ID),
+			Name: RequiredStringValue(project.Name),
+			URL:  RequiredStringValue(project.URL),
+		}
+	}
+	resourceModel.Projects = projects
+
+	return resourceModel
+}
+
+func (r *integrationGoogleCloudResource) Metadata(ctx context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_integration_google_cloud"
+}
+
+func (r *integrationGoogleCloudResource) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		MarkdownDescription: "Google Cloud Integration resource",
+
+		Attributes: map[string]schema.Attribute{
+			"aliases": schema.ListAttribute{
+				Description: "All of the aliases attached to the integration.",
+				Computed:    true,
+				ElementType: types.StringType,
+			},
+			"client_email": schema.StringAttribute{
+				Description: "The service account email OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"private_key": schema.StringAttribute{
+				Description: "The private key for the service account that OpsLevel uses to access the Google Cloud account.",
+				Required:    true,
+				Sensitive:   true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
```

Comment:
I'm confused why this requires replace given - https://jk-labs.slack.com/archives/C05RNQ6FD8U/p1723587269540589

Reply:
Removed this - should not be here.